.github/workflows/flyway-migrations.yml

```diff
@@ -46,7 +46,7 @@ jobs:
           mkdir ~/.ssh
           ssh-keyscan -H ${{ secrets.BASTION_HOST }} >> ~/.ssh/known_hosts
           eval `ssh-agent -s`
-          ssh -fN -v -L 5432:${{ env.CREDENTIALS_HOST }} ${{ secrets.BASTION_USER }}@${{ secrets.BASTION_HOST }} -i ssh_key.pem &
+          ssh -fN -v -L 5432:${{ env.CREDENTIALS_HOST }}:5432 ${{ secrets.BASTION_USER }}@${{ secrets.BASTION_HOST }} -i ssh_key.pem &
```

Comment:
Specifying the port twice?

Reply:
That is the syntax of the ssh tunneling command: https://www.ssh.com/academy/ssh/tunneling-example. It means that we want to forward the remote post 5432 to the local one.

Reply:
Oh very cool, something to read up on!